### PR TITLE
Flag gevent

### PR DESCRIPTION
**Pull request by ametaireau**
@leplatrem r?

### CODE REVIEW

**Comment by ametaireau** [comment]
I've started working on a test, but imports seem to not like it:

```py
import mock

from readinglist import patch_gevent
from .support import unittest


class InitialisationTest(unittest.TestCase):

    @mock.patch('readinglist.gevent')
    def test_gevent_monkeypatches_when_enabled(self, gevent_mock):
        with mock.patch('readinglist.gevent.monkey') as gevent_monkey:
            with mock.patch('readinglist.psycogreen.gevent') as psyco_monkey:
                patch_gevent({'readinglist.gevent_enabled': True})
               // Check mocks have been called.

```

File: docs/installation.rst [hunk 1]
```diff
@@ -199,13 +199,34 @@ On the first app run, the tables and objects are created.
...
-its path in the ``READINGLIST_INI`` environment variable.
+To use a different ini file, the ``READINGLIST_INI`` environment variable
+should be present with a path to it.
+
+
+Running with gevent
+-------------------
+
+It is possible to use `gevent <https://gevent.org>`_. To do so, the following
+configuration should be used.
```
Comment (leplatrem) [inline docs/installation.rst]: to use gevent, by enabling it in the configuration:

File: docs/installation.rst [hunk 2]
```diff
@@ -199,13 +199,34 @@ On the first app run, the tables and objects are created.
...
+      readinglist.gevent_enabled = True
+
+Gevent and psycogreen should be installed in the virtualenv for it to work
+properly::
+
+    .venv/bin/pip install gevent psycogreen
+
+:note:
+
+    Gevent support is known to have issues with Python 3, and as such it
```
Comment (leplatrem) [inline docs/installation.rst]: and as such, it

**Comment by ametaireau** [comment]
updated.

**Comment by ametaireau** [comment]
Tests are red because of the fix @leplatrem is working on.

**Comment by leplatrem** [comment]
Did you really update it ?

**Pull request merged**